tests/Validator/PossibleTypeExtensionsTest.php

```diff
@@ -0,0 +1,437 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GraphQL\Tests\Validator;
+
+use GraphQL\Type\Schema;
+use GraphQL\Utils\BuildSchema;
+use GraphQL\Validator\Rules\PossibleTypeExtensions;
+
+class PossibleTypeExtensionsTest extends ValidatorTestCase
```

Comment:
```suggestion
final class PossibleTypeExtensionsTest extends ValidatorTestCase
```

Reply:
Again, if this is a rule now,it would be great to enforce it with php-cs-fixer/phpstan.

Reply:
The fixer for this does not allow scoping this rule to directories. I don't want final classes in the implementation.